content/post/proxysql-one-2022.06.01/index.md

```diff
@@ -0,0 +1,536 @@
+      kind: MySQL
+      name: mysql-server
+    replicas: 3
+  tls:
+    issuerRef:
+      apiGroup: cert-manager.io
+      kind: Issuer
+      name: my-issuer
+    certificates:
+      - alias: server
+        subject:
+          organizations:
+            - kubedb:server
+        dnsNames:
+          - localhost
+        ipAddresses:
+          - "127.0.0.1"
+  terminationPolicy: WipeOut
+```
+* Now apply the yaml and wait for ProxySQL to be ready.
+```shell
+~$ kubectl apply -f proxysql.yaml
+proxysql.kubedb.com/proxy-mysql-server created
+
+~$ kubectl get pods -n demo | grep proxy
+proxy-mysql-server-0            1/1     Running   0          3m
+proxy-mysql-server-1            1/1     Running   0          3m
+proxy-mysql-server-2            1/1     Running   0          3m
+
+~$ kubectl get proxysql -n demo                                                                                                         
+NAME                 VERSION   STATUS   AGE
+proxy-mysql-server   2.3.2     Ready    4m
+
+```
+
+We are now ready to test our ProxySQL functionalities. We will follow the below steps to check our functionalities, <br>
+* [Create a test user, test database and table in MySQL server](#create-test-elements)
+* [Put an entry for the test user in ProxySQL server](#user-entry)
+* [Send load over the ProxySQL cluster as the test user](#send-load)
+* [Check the load distribution over the ProxySQL cluster](#check-load-distribution)
+* [Check TLS on the ProxySQL backend and frontend connection](#check-tls)
+
+
+#### <a id="create-test-elements"></a>Create test user, test database and table in MySLQ server
+* Exec into the MySQL primary pod with the following command, we will end up with something like this,
+```shell
+~ $ kubectl exec -it -n demo mysql-server-0 -- bash                            
+Defaulted container "mysql" out of: mysql, mysql-coordinator, mysql-init (init)
+root@mysql-server-0:/#
+```
+* Log into the MySQL console with root password.
+```shell
+root@mysql-server-0:/# mysql -uroot -p$MYSQL_ROOT_PASSWORD
+mysql: [Warning] Using a password on the command line interface can be insecure.
+Welcome to the MySQL monitor.  Commands end with ; or \g.
+Your MySQL connection id is 639
+Server version: 5.7.36-log MySQL Community Server (GPL)
+
+Copyright (c) 2000, 2021, Oracle and/or its affiliates.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+mysql>
+```
+* Let's get the current users.
+```shell
+mysql> select user from mysql.user;
++---------------+
+| user          |
++---------------+
+| proxysql      |
+| repl          |
+| root          |
+| mysql.session |
+| mysql.sys     |
+| root          |
++---------------+
+7 rows in set (0.00 sec)
+```
+note : the use `proxysql` is the monitor user for ProxySQL created by KubeDB operator
```

Comment:
note isn't clear
and use '>' that symbol before any kind of note, Like >Note:

Reply:
fixed